src/style.css

```diff
@@ -1,3 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  color: #000;
+  font-family: roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  height: 60px;
+  position: relative;
+  padding: 0 50px;
+  background-color: #fff;
+  text-transform: uppercase;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: inherit;
+  height: 100%;
+  display: flex;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

Comment:
These selectors divide the block nav selectors ( `nav_link` and `nav`)
You can put it before or after the nav block, and then all navigation selectors be together, in one place.